app/src/main/java/com/eveningoutpost/dexdrip/UtilityModels/BgSendQueue.java

```diff
@@ -39,6 +40,10 @@
     @Column(name = "operation_type")
     public String operation_type;
 
+    private static Context mContext = Home.getContext();
```

Comment:
gotcha gotcha (and yeah, sorry github is also awful on phones)

If we are worried about creating too many we should either have it be a singleton or a service, either would be fine with me. Im more of a fan of services as they feel more android-ish but they seem to have more resource overhead